functions/Set-DbaAgReplica.ps1

```diff
@@ -89,6 +93,18 @@ function Set-DbaAgReplica {
         PS C:\> Get-DbaAgReplica -SqlInstance sql2016 | Out-GridView -Passthru | Set-DbaAgReplica -BackupPriority 5000
 
         Sets the backup priority to 5000 for the selected availability groups.
+
+    .EXAMPLE
+        PS C:\> Get-DbaAgReplica -SqlInstance sql2016 -Replica Replica1 |
+        >> Set-DbaAgReplica -ReadOnlyRoutingList Replica2,Replica3;
+
```

Comment:
also, please note that semicolons are not required and we prefer not to use them, as it keeps the code tidier.